x-pack/docs/en/rest-api/security/create-cross-cluster-api-key.asciidoc

```diff
@@ -3,7 +3,241 @@
 === Create Cross-Cluster API key API
 
 ++++
-<titleabbrev>Create Cross-Cluster API key</titleabbrev>
+<titleabbrev>Create Cross-Cluster API key API</titleabbrev>
 ++++
 
-TODO: Placeholder
+Creates an API key of the `cross_cluster` type for the API key based remote cluster access.
```

Comment:
Once we add relevant docs for "API key based remote cluster model", we can link to it from here to give more context. For now, this remains to be just plain text.